### ISSUE TITLE
[FEAT] Better line spacing for bigger fontsize or Headers

### ISSUE THREAD

**Issue by dummifiedme** [st3v3nmw/obsidian-spaced-repetition#248]
Better line spacing for bigger fontsize or Headers

Example card: 
Notice the first "Front" line. 
![image](https://user-images.githubusercontent.com/68785729/129474854-d4fd0140-f1b3-4da3-a15a-f00a45e59fda.png)

**Comment by st3v3nmw** [comment]
Hmm, I thought the headers aren't part of the card text itself & should appear as part of the context, otherwise it's redundant.

**Comment by dummifiedme** [comment]
Yes, I would be removing the context. Most of my notes are actually just cards.
